src/zeit/content/image/interfaces.py

```diff
@@ -58,8 +59,10 @@ class IImageMetadata(zope.interface.Interface):
 
     copyrights = zope.schema.Tuple(
         title=_("Copyrights"),
-        default=((u'©', None, None, None, False),),
+        default=((u'', None, None, None, False),),
         missing_value=(),
+        min_length=1,
+        max_length=1,
```

Comment:
Sorry falls das eine total dämliche Frage ist, aber: Können wir dann nicht das `schema.Tuple` einfach komplett raushauen? Weil auf Anhieb seh ich nicht, wofür eine 1-elementige Liste gut sein soll. :)

Reply:
Hatten wir auch überlegt. Ist ja jetzt etwas merkwürdig moddeliert. Der Grund warum wir das so gemacht haben, ist vor allem, dass wir uns dann BW-Compat Code sparen können.

Reply:
Hm... Im DAV ist das so oder so ein xmlpickle, d.h. das kann man dann behandeln, wie man will. Insofern ist glaub 1-2 Zeilen [hier in der Ecke](https://github.com/ZeitOnline/zeit.content.image/blob/master/src/zeit/content/image/metadata.py#L37) "wenn es eine Liste von Tupel ist, dann nimm nur das erste Element, danke" glaub ich das kleinere Übel, als im Friedbert etc.pp. überall ständig das ein(zig)e Element auswickeln zu müssen, oder?